README.md

```diff
@@ -52,36 +52,32 @@
 ## Categories
 
 #### GitHub Actions 🤖
-- [Abhishek Naidu](https://github.com/abhisheknaiidu/abhisheknaiidu)
+- [Abhishek Naidu](https://github.com/abhisheknaiidu/abhisheknaiidu) 
```

Comment:
why such a change?

Reply:
Oh sorry, didn't see it was profile of the repo owner! My bad!

Reply:
> why such a change?

Oh sorry, didn't see it was profile of the repo owner! My bad!